dotnet/test/common/BiDi/Network/NetworkEventsTest.cs

```diff
@@ -132,11 +132,8 @@ public async Task CanListenToFetchError()
 
         await using var subscription = await context.Network.OnFetchErrorAsync(tcs.SetResult);
 
-        try
-        {
-            await context.NavigateAsync("https://not_a_valid_url.test", new() { Wait = ReadinessState.Complete });
-        }
-        catch (Exception) { }
+        var navigateTask = context.NavigateAsync("https://not_a_valid_url.test", new() { Wait = ReadinessState.Complete });
+        await ((Task)navigateTask).ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing);
```

Comment:
```suggestion
        await (context.NavigateAsync("https://not_a_valid_url.test", new() { Wait = ReadinessState.Complete }) as Task)
            .ConfigureAwait(ConfigureAwaitOptions.SuppressThrowing);
```

I think writing it as one statement is cleaner, removing extra variable.

Reply:
What do you think of this now? The `as` operator makes me uncomfortable when we know the exact type and don't handle `null`s

Reply:
Actually I don't like this style in tests project, it is harder to understand and maintain. As for me `try/catch` is simpler, we can easily extend of `catch` cases. Probably I am old-school.

But, I see benefit using it in performance critical code execution. There, it would be good.

@YevgeniyShunevych what do you think?

Reply:
> Modern NUnit supports delegates returning values.
I was actually eyeing how much effort it would be to update NUnit to 4.x 😄 

> Prefer TypeOf instead of InstanceOf to be exact.
`InstanceOf` would maintain the existing `catch` behavior, Let me switch to `TypeOf` and make sure nothing breaks